UPGRADE.md

```diff
@@ -26,6 +26,9 @@
 1. Constructor of `Sylius\RefundPlugin\Entity\CustomerBillingData` has been removed and now `CustomerBillingData` entity
    is created by `Sylius\RefundPlugin\Factory\CustomerBillingDataFactory`.
 
+1. Constructor of `Sylius\RefundPlugin\Entity\ShopBillingData` has been removed and now `ShopBillingData` entity
```

Comment:
`Constructor of `Sylius\RefundPlugin\Entity\ShopBillingData` has been removed` nope 💃